### PR TITLE
Fixes aggregating metrics for profile

### PR DESCRIPTION
**Pull request by ajainarayanan**

### CODE REVIEW

**Comment by bdmogal** [comment]
@ajainarayanan can you describe this issue a little more?

**Comment by ajainarayanan** [comment]
@bdmogal Sure. 
- We used to query profile metrics for 24hrs and for overall time. 
- While querying for 24hours there are times where the backend will return a timeseries since aggregation on 24hr window is not possible.
- So UI has to reduce the time series by aggregating it in UI. 
- In the reduce step in UI there was an issue where I was not providing the initial value which resulted in a state like this
screen shot 2018-07-20 at 9 56 16 am

This fix is to prevent this from happening.

**Comment by bdmogal** [comment]
thanks @ajainarayanan

File: cdap-ui/app/cdap/components/Cloud/Profiles/Store/ActionCreator.js [hunk 1]
```diff
@@ -207,7 +207,7 @@ export const getProfiles = (namespace) => {
                   if (metric.data.length === 1) {
                     metricValue = metric.data[0].value;
                   } else {
-                    metricValue = metric.data.reduce((prev, curr) => prev + curr.value);
+                    metricValue = metric.data.reduce((prev, curr) => prev + curr.value, metric.data[0].value);
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Cloud/Profiles/Store/ActionCreator.js]: wouldn't this double the the first value? Shouldn't this start from 0?

File: cdap-ui/app/cdap/components/Cloud/Profiles/Store/ActionCreator.js [hunk 2]
```diff
@@ -127,7 +127,7 @@ export const fetchAggregateProfileMetrics = (namespace, profile, extraTags) => {
               if (metric.data.length === 1) {
                 metricValue = metric.data[0].value;
               } else {
-                metricValue = metric.data.reduce((prev, curr) => prev + curr.value);
+                metricValue = metric.data.reduce((prev, curr) => prev + curr.value, metric.data[0].value);
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/Cloud/Profiles/Store/ActionCreator.js]: Just want to confirm, it's `prev + curr.value`, not `prev.value + curr.value`?

Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/Cloud/Profiles/Store/ActionCreator.js]: No its just adding numbers.

**Review by tonybach — APPROVED** [review]
Code LGTM 👍

**Review by elfenheart — APPROVED** [review]
👍  LGTM

**Review by bdmogal — APPROVED** [review]

**Pull request merged**